### DISCUSSION TITLE
vuepress-plugin-vssue-next-compat

### DISCUSSION THREAD

**Discussion by laoergege** [vuepress/core#939] [Show and tell]
I write a vssue plugin for vuepress-next. Everyone can try 👻

[vuepress-plugin-vssue-next-compat ](https://github.com/laoergege/laoergege-blog/packages/1480309)

**Comment by laoergege** [comment]
Update!
Add an "onBeforeOauth hook" which you can modify the OAuth URL is running before vssue redirects to the OAuth URL.

I found that the redirect URL of the GitHub callback cannot contain the Chinese path!  So I add a new API in which you can modify the redirect_uri query param.

**Comment by Mister-Hope** [comment]
Do you have any interests adding this work to `vuepress-plugin-comment2`?

Also, the `CHANGELOG.md` link is broken.

**Reply by laoergege** [reply]
Thanks for the reminder, I plan to finish writing it and test it on my blog first, then try to support vuepress-plugin-comment2.
